Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -95,6 +95,17 @@ public CorrelationIdLookupHelper(string endpointAddress)
         }
 
         /// <summary>
+        /// Gets the empty correlation id based on the correlation id format. This is sent as a header to indicate that we are dealing with a tracked component, but we could not fetch the appId just yet.
+        /// </summary>
+        public string EmptyCorrelationId
+        {
+            get
+            {
+                return string.Format(CultureInfo.InvariantCulture, CorrelationIdFormat, string.Empty);
+            }
+        }
```

Comment:
nit: Will this string.Format happen every-time it is read? Although won't be too much gain, I suggest we make this a static read-only field.